easybuild/easyconfigs/o/OpenCV/OpenCV-3.2.0-CrayGNU-17.08-cuda-8.0-python3.eb

```diff
@@ -0,0 +1,85 @@
+# @author: Guilherme Peretti-Pezzi (CSCS), mschoengens
+
+easyblock = 'CMakeMake'
+
+name = 'OpenCV'
+version = '3.2.0'
+
+py_maj_ver = '3'
+py_min_ver = '5'
+py_rev_ver = '2'
+
+req_py_majver = int(py_maj_ver)
+req_py_minver = int(py_min_ver)
+
+cudamajor = '8.0'
+cudaminor = '61_2.4.3-6.0.4.0_3.1__gb475d12'
+cudaver = "%s.%s" % (cudamajor, cudaminor)
+
+versionsuffix = '-cuda-%s-python%s' % (cudamajor, py_maj_ver)
+
+homepage = 'http://opencv.org/'
+description = """OpenCV (Open Source Computer Vision Library) is an open source computer vision
+ and machine learning software library. OpenCV was built to provide
+ a common infrastructure for computer vision applications and to accelerate
+ the use of machine perception in the commercial products."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'pic': True, 'verbose': False}
+
+source_urls = ['https://github.com/opencv/opencv/archive/']
+sources = ['%(version)s.zip']
+
+builddependencies = [
+    ('CMake', '3.10.1', '', True),
+]
+
+dependencies = [
+    ('cray-python/17.06.1', EXTERNAL_MODULE),
+    ('zlib', '1.2.11'),
+    ('FFmpeg', '2.8.7'),
+    ('libjpeg-turbo', '1.5.2'),
+    ('libpng', '1.6.23'),
```

Comment:
I've missed this one (it should be 1.6.30).

Reply:
done.